apps/web/lib/hooks/settings/platform/oauth-clients/useOAuthClientWebhooks.ts

```diff
@@ -0,0 +1,84 @@
+import { useMutation, useQuery } from "@tanstack/react-query";
+
+import type { ApiSuccessResponse } from "@calcom/platform-types";
+import type { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+type Input = {
+  active: boolean;
+  subscriberUrl: string;
+  triggers: WebhookTriggerEvents[];
+  secret?: string;
+  payloadTemplate?: string;
+};
+
+type Output = {
```

Comment:
For extra safety we could define OutputSchema using zod instead and then parse `res.json()` and make return Output[] inferred from zod type.